src/tcgwars/logic/impl/gen4/MysteriousTreasures.groovy

```diff
@@ -1660,32 +1672,39 @@ public enum MysteriousTreasures implements LogicCardInfo {
           weakness G, PLUS10
           resistance L, MINUS20
           move "Mud Spit", {
-            text "Flip a coin. If heads, this attack does 10 damage to each of your opponent’s Pokémon."
+            text "Flip a coin. If heads, this attack does 10 damage to each of your opponent’s Pokémon. (Don’t apply Weakness and Resistance for Benched Pokémon.)"
             energyCost F
             attackRequirement {}
             onAttack {
-              damage 0
+              flip{ opp.all.each{ damage 10, it } }
             }
           }
 
         };
       case BIDOOF_73:
         return basic (this, hp:HP060, type:COLORLESS, retreatCost:2) {
           weakness F, PLUS10
-          move "", {
-            text "Any damage. "
-            energyCost ()
-            attackRequirement {}
-            onAttack {
-              damage 0
+          //Wacan Berry: Any damage done to Bidoof by attacks from [L] Pokémon is reduced by 20 (after applying Weakness and Resistance).
+          customAbility {
+            delayedA {
+              before APPLY_ATTACK_DAMAGES, {
+                if(ef.attacker.owner != self.owner && (ef.attacker.types.contains(R) || ef.attacker.types.contains(W))) {
+                  bg.dm().each{
+                    if(it.to == self && it.notNoEffect && it.dmg.value) {
+                      bc "Chesto Berry -20"
```

Comment:
I'm guessing this whole thing was copy and pasted from another card and you forgot to change it :laughing:

Reply:
Replied above 😝